system-addon/content-src/components/TopSites/TopSite.jsx

```diff
@@ -386,10 +383,11 @@ export class _TopSiteList extends React.PureComponent {
       dispatch: props.dispatch,
       intl: props.intl
     };
+    let holeIndex = 0;
```

Comment:
Forgot the comment about why this was necessary

Reply:
doh! I knew something was missing. I kept rereading the old comments.